app/src/main/java/org/openobservatory/ooniprobe/common/MapUtility.java

```diff
@@ -0,0 +1,10 @@
+package org.openobservatory.ooniprobe.common;
+
+import java.util.HashMap;
+
+public class MapUtility {
+    //Class replacement for HashMap's function getOrDefault that doesn't work in android < N
```

Comment:
Can we put the actual version of Android where it doesn't work?

Otherwise looks good to merge.

Reply:
I would leave it here so we know when we can remove this class.